README.md

```diff
@@ -1,41 +1,92 @@
-with URL `https://mc-stan.org/docs`.
-The publishing strategy is to serve the contents of the directory `docs` on branch `master`.
-The `docs` directory contains an empty file named `.nojekyll` so that GitHub will treat the contents
-as pre-generated HTML instead of trying to run [jekyll](https://jekyllrb.com).
 
-## Directory Structure
+## Repository directory structure
 
-* `src` : directory of source files for Stan and CmdStan guides and reference manuals, each in its own named subdirectory.
+* `src` : directory of source files for Stan and CmdStan guides and reference manuals, each in its own named subdirectory:
+    + `src/cmdstan-guide` - CmdStan Guide
+    + `src/functions-reference` - Stan Functions Reference
+	+ `src/reference-manual` - Stan Reference Manual
+	+ `src/stan-users-guide` - Stan Users Guide
 
 * `docs`: the directory `docs` on branch `master` is the [publishing source](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/) for the project pages site.  Whenever a verified member of the Stan organization pushes to `docs` on branch `master`,
 GitHub (re)builds and (re)deploys the website.
-  
-* `build.py`: python script which compiles markdown files under `src` to html and pdf and populates the `docs` dir with the generated documentation.
 
-  + arg 1: MAJOR Stan version, required (expecting number, should be positive int)
-  + arg 2: MINOR Stan version, required (expecting number, should be positive int)
-  + arg 3: name of document (optional - will build entire docset)
-  + arg 4: output format, either "html" or "pdf" (optional - default builds both html and pdfs)
-  
-* `LICENSE`: licensing terms.
+## Documentation toolset
+
+The documentation source files are written in [Rmarkdown](https://rmarkdown.rstudio.com)
+and the RStudio's [bookdown package](https://github.com/rstudio/bookdown) converts these to HTML and pdf.
+**Required bookdown version:** 2.23 or higher, cf [the bookdown changelog](https://github.com/rstudio/bookdown/blob/main/NEWS.md#changes-in-bookdown-version-023).
+Reported via [issue #380](https://github.com/stan-dev/docs/issues/380).
+
+
+The conversion engine is [Pandoc](https://pandoc.org).  It is bundled with RStudio.
+To use the build scripts to build the docset,
+you might need to [install Pandoc](https://pandoc.org/installing.html) separately.
+
+To build the pdf version of the docs, you will need to [install LaTeX](https://www.latex-project.org/get/) as well.
+The Stan documentation uses the [Lucida fonts](https://www.pctex.com/Lucida_Fonts.html),
+which must be [installed manually](https://tex.stackexchange.com/questions/88423/manual-font-installation).
+
+
+## Scripts to build and maintain the docset
+
+**`build.py`**
+
+The program `build.py` convert the markdown files under `src` to html and pdf and populates the `docs` dir with the generated documentation.
+Requires Python 3.5 or higher, due to call to `subproces.run`.
```

Comment:
While `subprocess.run` was added in 3.5, the `capture_output` kwarg wasn't until 3.7 -- I think that's probably fine, but we should correct the minimum version here.

Reply:
also checking in script itself.